### ISSUE TITLE
phase plate data error modeling

### ISSUE THREAD

**Issue by craigyk** [cryoem-uoft/cryosparc-issues#113]
Not sure if this is an actual issue, but thought I'd share my results processing my phase plate dataset.  My first run through got to 3.5Å, so pretty good... but I did notice during the refinement that the modeled errors at low resolution are much higher than for conventional data, and also the initial estimate underestimates the error rather than overestimates and approaches.  For processing this data in Relion, I often choose to ignore the data before the first CTF zero. I know Radostin sometimes also high-pass filters his phase-plate data to 60Å to accomplish a similar effect (reduce very low-res influence).

![screen shot 2016-12-21 at 1 21 46 pm](https://cloud.githubusercontent.com/assets/1326103/21406492/ce8bc8d4-c780-11e6-8cd2-f929c591ec5a.png)

**Comment by mbrubake** [comment]
Yeah, I had noticed that as well when I was testing.  I haven't seen it cause problems yet but it's something to keep an eye on.

We probably need to tweak the way the noise model adapts a bit.  I'll look into it when I get a spare cycle.

**Comment by mbrubake** [comment]
Also how does 3.5Å compare to RELION on that dataset?

**Comment by craigyk** [comment]
The best I got with Relion 2 was actually 3.1 but I haven't spent as much time tweaking things in CryoSparc as I have in Relion.  The maps themselves look very comparable (side chains, etc), so maybe this is mostly differences due to masking and FSC calc?  I haven't yet done a careful side-by-side comparison.

**Comment by mbrubake** [comment]
Yes, I suspect masking (and maybe this noise issue) could be at plat.  If you get a chance, can you also post the last noise model plot from refinement?  I'm curious what the model looked like at convergence.

**Comment by craigyk** [comment]
I just noticed my dynamic masking options were way too aggressive on this run, I was probably still experimenting, but it looks like a different mask is generated for FSC calculation anyways, and those masks look pretty good.

![585f19c58a72810fd5ba7997](https://cloud.githubusercontent.com/assets/1326103/22119506/bab726d4-de30-11e6-9d67-5aab83dc36fc.png)
![585f19c78a72810fd5ba799e](https://cloud.githubusercontent.com/assets/1326103/22119505/bab69368-de30-11e6-846e-6c29fcaa77b3.png)
![585f19c98a72810fd5ba79aa](https://cloud.githubusercontent.com/assets/1326103/22119507/bab9c6be-de30-11e6-837f-fbcafac1f2ea.png)
